music21/musicxml/m21ToXml.py

```diff
@@ -3892,15 +3891,27 @@ def noteToNotations(self, n, notFirstNoteOfChord=False, chordParent=None):
                 else:
                     notations.append(mxExpression)
 
-            for artObj in chordOrNote.articulations:
-                if 'TechnicalIndication' in artObj.classes:
-                    if mxTechnicalMark is None:
-                        mxTechnicalMark = Element('technical')
-                    mxTechnicalMark.append(self.articulationToXmlTechnical(artObj))
-                else:
-                    if mxArticulations is None:
-                        mxArticulations = Element('articulations')
-                    mxArticulations.append(self.articulationToXmlArticulation(artObj))
+        # apply all articulations apart from fingerings only to first note of chord
+        from music21.articulations import Fingering
+        applicableArticulations = []
+        fingeringNumber = 0
+        for a in chordOrNote.articulations:
+            if type(a) is Fingering:
```

Comment:
do not do type(x) checks.  use isinstance -- otherwise subclasses fail.  But better still is use the music21.base.Music21Object.classes or classSet interface -- it is much faster than isinstance.